src/OpenCV/ImgProc/GeometricImgTransform.hsc

```diff
@@ -297,3 +298,37 @@ getRotationMatrix2D center angle scale = unsafeCoerceMat $ unsafePerformIO $
   where
     c'angle = realToFrac angle
     c'scale = realToFrac scale
+
+{- | Applies a generic geometrical transformation to an image.
+
+<http://docs.opencv.org/3.0-last-rst/modules/imgproc/doc/geometric_transformations.html#remap>
+-}
+remap
+    :: Mat ('S [inputHeight, inputWidth]) inputChannels inputDepth
+    -> Mat ('S [outputHeight, outputWidth]) ('S 2) ('S Float)
```

Comment:
Could you add some documentation to the various parameters? Like how this 2-channel matrix encodes (x,y) points.

Even without documentation it is useful to put at least one ` -- ^` after a parameter. This prevents Haddock from generating extremely long unreadable lines.